HelpSource/Classes/ScopeOut.schelp

```diff
@@ -1,11 +1,11 @@
 class:: ScopeOut
-summary:: FIXME: ScopeOut purpose.
+summary:: Used to communicate with link::CLasses/SCScope::
 categories::  UGens>Buffer, UGens>Undocumented
 
 
 Description::
 
-FIXME: ScopeOut description.
+This is used to communicate with link::Classes/SCScope::, but only when running the server in internal mode.
```

Comment:
ScopeOut works only with internal server.
Suggestion (feel free to adjust):

This is used by the internal server to communicate with link::Classes/SCScope::.